Triphouse-project/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
Are you open your page in browser ?
![2021-04-08_22h34_28](https://user-images.githubusercontent.com/19935417/114085869-9b31c000-98ba-11eb-8254-75067e6a3949.png)
Please check your code, review will be after changes.

Reply:
I check my code and fix